lib/syskit/network_generation/dataflow_dynamics.rb

```diff
@@ -618,9 +627,27 @@ def policy_for(
                           "#{sink_port_m.required_connection_type} " \
                           "on #{sink_port}"
                 end
+                policy
+            end
+
+            def policy_for(
+                source_task, source_port_name, sink_port_name, sink_task,
+                fallback_policy, explicit_policy: nil
+            )
+                computed_policy = {}
+                explicit_policy = {} unless explicit_policy
```

Comment:
```suggestion
                fallback_policy, explicit_policy: {}
            )
                computed_policy = {}
```

You mentioned making this a kw arg due to not wanting to fix the policy_for call alls over the code, but you can make this a positional argument with a default value as well, that would work just like the kw arg worked.

Reply:
This. I think that's the last change.